src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    protected final String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
```

Comment:
```suggestion

```
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced)
Don’t use abstract classes to set behaviour for classes
Abstract classes and interfaces have different use cases. Try to figure out when to use both in this task by yourself. If you’re blocked [this](https://stackoverflow.com/a/479168) may give you a hint.

Reply:
So I need to remove "abstract" modificator from getArea method, create default method body and than override method in classes that use it ? 
```suggestion

    public double getArea(){
       return 0;
    }
```

Reply:
This method in Figure file is unnecessary, because I used it when I was building main logic. When I opened file with common mistakes, I created an interface with method getArea.